pages/docs/2_modify/1_getting_started/3_custom_init.md

```diff
@@ -138,17 +135,21 @@ For this, add the following **scripts to our `package.json`** file:
     "prepublishOnly": "npm run build",
     "build:engine": "comunica-compile-config config/config-default.json > engine-default.js",
     "build:lib": "tsc",
-    "build": "npm run build:lib && npm run build:engine",
-    "postinstall": "npm run build"
```

Comment:
"postinstall" is wrong indeed, but we'll probably want to replace it by "prepare".